index.js

```diff
@@ -18,4 +18,247 @@ const openMenu = document.querySelector('.button');
+  image: '<img class="yoga-im" src="./desktopimg/backgrounds/card2.png" alt="project1">',
+  technologies: ['html', 'bootstrap', 'ruby'],
+  link: link.innerHTML,
+  source: source.innerHTML,
+},
+{
+  name: 'Website Protfolio',
+  description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+  image: '<img class="yoga-im" src="./desktopimg/backgrounds/card3.png" alt="project1">',
+  technologies: ['html', 'bootstrap', 'ruby'],
+  link: link.innerHTML,
+  source: source.innerHTML,
+},
+{
+  name: 'Profesional Art Printing Data More',
+  description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+  image: '<img class="yoga-im" src="./desktopimg/backgrounds/card4.png" alt="project1">',
+  technologies: ['html', 'bootstrap', 'ruby'],
+  link: link.innerHTML,
+  source: source.innerHTML,
+},
+{
+  name: 'Data Dashboard Healthcare',
+  description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+  image: '<img class="yoga-im" src="./desktopimg/backgrounds/card5.png" alt="project1">',
+  technologies: ['html', 'bootstrap', 'ruby'],
+  link: link.innerHTML,
+  source: source.innerHTML,
+},
+{
+  name: 'Website Protfolio',
+  description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+  image: '<img class="yoga-im" src="./desktopimg/backgrounds/card6.png" alt="project1">',
+  technologies: ['html', 'bootstrap', 'ruby'],
+  link: link.innerHTML,
+  source: source.innerHTML,
+},
+];
+
+function works(html) {
+  const template = document.createElement('template');
+
+  template.innerHTML = html.trim();
+
+  return template.content;
+}
+
+for (let i = 0; i < arr.length; i += 1) {
+  if (i === 0) {
+    const card1 = works(`<div class="title">
+  <h2 id ="heading">My Recent Works</h2>
+  <div class="empty"></div>
+  </div>
+  </div>
+  <div class="work-grid">
+  <div class="desk-work">
+  <a id="yoga" href="#">${arr[i].image}</a>
+  <div class="desk-yoga">
+  <h3 id="heading3" class="heading3">${arr[i].name}</h3>
+  <p class="framepara">${arr[i].description}</p>
+  <div id="tags">
+      <ul class="tagsul">
+          <li class="tags1" id="tag1">${arr[i].technologies[0]}</li>
+          <li class="tags1" id="tag2">${arr[i].technologies[1]}</li>
+          <li class="tags1" id="tag3">${arr[i].technologies[2]}</li>
+          <li class="tags1" id="tag4">${arr[i].technologies[3]}</li>
+      </ul>
+      <button id="project-button" class= "project-button">See Project</button>
+  </div>
+  </div>
+</div>
+<div class = "frame">
+<div class = "cards"></div>
+</div>
+`);
+    document.body.querySelector('#portfolio').appendChild(card1);
+  } else if (i > 0) {
+    const cards = works(`<div class="card">
```

Comment:
[OPTIONAL] The cards actually are the same from an objective point of view, their differences are in the array, so I do not see the need for an else block. Since you are using a loop, `i` refer to each object in the array, you could have used the dot notation to access whatever you want. Consider this suggestion as it will improve the code even more.

Reply:
I put it like that because one project looks different and rest 6 card are in grid.